app/views/steps/solicitor/contact_details/edit.html.erb

```diff
@@ -1,20 +1,20 @@
 <% title t('.page_title') %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
     <%= step_header %>
 
-    <h1 class="heading-xlarge gv-u-heading-xxlarge">
-      <%=t '.heading', name: @form_object.full_name %>
+    <h1 class="govuk-heading-xl">
+      <%= t '.heading', name: @form_object.full_name %>
     </h1>
 
-    <%= step_form @form_object do |f| %>
-      <%= f.text_area :address, size: '40x4', class: 'form-control-3-4' %>
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_text_area :address %>
 
-      <%= f.text_field :dx_number, class: 'narrow' %>
-      <%= f.email_field :email %>
-      <%= f.text_field :phone_number, class: 'narrow' %>
-      <%= f.text_field :fax_number, class: 'narrow' %>
+      <%= f.govuk_text_field :dx_number, class: 'narrow' %>
```

Comment:
These need to use the new format `width: 'one-half'` instead of applying directly the class narrow.
For the phone and fax numbers better to use `govuk_phone_field` as per https://govuk-form-builder.netlify.app/form-elements/text-field/

Reply:
Thanks, solved